web/utils.js

```diff
@@ -439,3 +439,173 @@ function checkEnter(event) {
         $("#sbox").submit();
     }
 }
+
+// Intelligence Window code starts from here
+document.onmousemove = function(event) {
+    if (typeof event !== 'undefined') {
+        document.intelliWindowMouseX = event.clientX;
+        document.intelliWindowMouseY = event.clientY;
+    }
+};
+
+$(document).keypress(function(e) {
+    if (document.activeElement.id === 'search' ||
+        typeof document.intelliWindow === 'undefined') {
+        return true;
+    }
+
+    if (e.which === 49) { // '1' pressed
+        if (document.intelliWindow.className === "intelli_window_style") {
+            hideIntelliWindow();
+        } else if (document.intelliWindow.className === "intelli_window_style_hide") {
+            showIntelliWindow();
+        }
+    }
+    if (e.which === 50) { // '2' pressed
+        var symbol = document.intelliWindow.symbol;
+        var highlighted_symbols_with_same_name = $("a").filter(function(index) {
+            return $(this).text() === symbol &&
+                $(this).css("background-color") === "gold";
+        })
+        if (highlighted_symbols_with_same_name.length === 0) {
+            highlightSymbol(symbol);
+        } else {
+            unhighlightSymbol(symbol);
+        }
+    }
+    return true;
+});
+
+function onMouseOverSymbol(symbol, symbolType) {
+    updateIntelliWindow(symbol, symbolType);
```

Comment:
1) I use it now with OnMouseOver and I must say that problems you describe are true, but not very big deal. I didn't have any big problems highlighting exactly what I want. 
Sometimes it is just blinking too much.
Still it can be disturbing for user not interested in highlighting.

So let's improve it a bit:
1a) OnMouseOver could be improved with short delay - mouse have to stop for ~1s on symbol to highlight it. Highlight is removed as soon as another symbol is highlighted.
I think this solves all problems.

2) I propose this solution because all users will be able to notice the feature without reading any documentation.
Unfortunately I'm sure that from all users of OpenGrok server I installed, I'm the only one who have ever read help page/release notes/documentation.